### PR TITLE
Fix reorder

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
Server Side Fix for https://github.com/symfony-cmf/tree-browser-bundle/issues/153

### CODE REVIEW

File: src/Controller/TreeController.php [hunk 1]
```diff
@@ -102,6 +102,10 @@ public function reorderAction(Request $request)
             return new JsonResponse(array('Parameters parent, dropped and target has to be set to reorder.'), Response::HTTP_BAD_REQUEST);
         }
 
+        if (in_array($position, array('over', 'child'))) {
+            return new JsonResponse(array('Can not resort into a collection.'), Response::HTTP_BAD_REQUEST);
```
Comment (greg0ire) [inline src/Controller/TreeController.php]: Is "resort" really what you mean here? http://www.wordreference.com/enfr/resort
the name of the action is "reorder" BTW

Comment (ElectricMaxxx) [inline src/Controller/TreeController.php]: i mean "reorder", and that should be done, when position is `before` or `after` a node. Reorder will interally set the position property of a node collection.

Comment (ElectricMaxxx) [inline src/Controller/TreeController.php]: now i see your issue, i am not that good in wording, even in my home language :-) So what do i mean, based on the description before?

Comment (greg0ire) [inline src/Controller/TreeController.php]: Then use reorder in the message please

Comment (ElectricMaxxx) [inline src/Controller/TreeController.php]: thanks. BTW: this route will be removed, when `resources-rest-bundle` provides a PATCH route  on our position.

**Comment by ElectricMaxxx** [comment]
i will load that code into our sandbox and see it working before merging and releasing.

**Comment by ElectricMaxxx** [comment]
i excluded that call on client side, but whanted to have the backend fail safe.

**Pull request merged**